app/src/androidTest/java/net/skyscanner/backpack/calendar/BpkCalendarTest.kt

```diff
@@ -170,6 +171,21 @@ class BpkCalendarTest : BpkSnapshotTest() {
       }
   }
 
+  @Test
+  @FlakyTest
```

Comment:
Would it make sense to write some detail on why is this flaky as it is defined here: https://developer.android.com/reference/android/support/test/filters/FlakyTest ?

Reply:
It's flaky because it fails on CI, we have a bunch of those cases and we need to figure out what's causing it. But for now we run them only locally and not on ci. Exactly why it's flaky we don't, it just times out on ci.